netmap/netmap_contract.go

```diff
@@ -200,73 +219,27 @@ func Update(script []byte, manifest []byte, data interface{}) {
 	runtime.Log("netmap contract updated")
 }
 
-// InnerRingList method returns a slice of structures that contains the public key of
-// an Inner Ring node. It should be used in notary disabled environment only.
-//
-// If notary is enabled, look to NeoFSAlphabet role in native RoleManagement
-// contract of the sidechain.
+// InnerRingList instantly panics because it was intended for legacy non-notary
+// deployments only.
 func InnerRingList() []common.IRNode {
-	ctx := storage.GetReadOnlyContext()
-	pubs := getIRNodes(ctx)
-	nodes := []common.IRNode{}
-	for i := range pubs {
-		nodes = append(nodes, common.IRNode{PublicKey: pubs[i]})
-	}
-	return nodes
+	// TODO: consider method removal
```

Comment:
Can be removed if we're sure notary-enabled node never calls them.

Reply:
Method docs require this. We'll definitely double-check that in `neofs-node`. Anyway, if some notary-adopted code calls the method, it'll encounter `unknown method` exception.